index.js

```diff
@@ -107,5 +146,13 @@ module.exports = function Addon(manifest) {
 		publishToDir(baseDir || './publish-'+manifest.id, manifest, handlers)
 	}
 
+	this.publishToWeb = function(addonUrl) {
+		if (!addonUrl || !addonUrl.includes('https://')) throw 'Please set a valid https url'
+
+		// Create a manifest url
+		manifestUrl = addonUrl.replace('https://', 'stremio://') + '/manifest.json';
```

Comment:
let's use `manifestUrl = addonUrl.replace('https://', 'stremio://');` here (without adding '/manifest.json' to the end), it can get confusing like this, i used a manifest url and ended up with `/manifest.json/manifest.json` at the end of it

Reply:
we can also check if the url does not include `/manifest.json` and throw an error for that